plugins/src/lib.rs

```diff
@@ -46,6 +46,7 @@ where
     rpcmethods: HashMap<String, RpcMethod<S>>,
     subscriptions: HashMap<String, Subscription<S>>,
     dynamic: bool,
+    nonnumericids: bool,
```

Comment:
The Rust plugin handles the `id` completely internally, so this is not needed at all (also why the warning is being returned during compilation).